docs/explanation/ssh.md

```diff
@@ -0,0 +1,111 @@
+(juju-ssh)=
+# Juju SSH
+
+Juju 3.6 introduces a new way to SSH into machines and Kubernetes units.
+
+Instead of connecting directly to machines, SSH connections are now routed through the Juju controller (or via JIMM if used). This centralization improves auditability and security, while keeping the user experience largely unchanged. 
+
+Users can continue to run `juju ssh` or use standard SSH clients.
+
+## Overview
+
+The Juju controller now runs an SSH server capable of terminating SSH connections
+and then forwards the traffic to the target machine or K8s unit.
+
+When a client attempts to SSH to a target unit/machine, the command will resemble the following,
+using OpenSSH syntax.
+
+```
+ssh -J <controller-address>:17022 ubuntu@<unit-number.app.modelUUID>
+```
+
+The ` -J` flag performs an "Jump" connection.
+This is a process that may be familiar to those who connect to machines through a jump/bastion host,
+effectively establishing two SSH connections to reach the target. Supported in some fashion by many SSH clients.
+
+The Juju controller terminates both connections, allowing it to inspect the traffic before
+forwarding it.
+
+## Security
+
+The main security enhancement of the SSH Proxy is the removal of direct SSH access to machines.
+
+Instead, connections are proxied through the Juju controller, which:
+
+- Limits the network surface exposed to the public.
+- Avoids multiple copies of a user's keys on each machine they want to access.
+- Allows the Juju controller to present stronger access controls beyond what SSH offers.
+- Enables better management of SSH session policies.
```

Comment:
in future, it will also allow for better audit logging giving admins visibility into what users are doing to their systems